src/webpack.js

```diff
@@ -27,6 +28,10 @@ class ReactLoadablePlugin {
     compiler.plugin('emit', (compilation, callback) => {
       const manifest = buildManifest(compiler, compilation);
       var json = JSON.stringify(manifest, null, 2);
+      const outputDirectory = path.dirname(this.filename);
+      if (!fs.existsSync(outputDirectory)) {
+        fs.mkdirSync(outputDirectory);
```

Comment:
Instead of checking if a directory exists and then creating it if it doesn't, it's better to just attempt to create the directory and protect against it throwing an error when it does exist. Otherwise you introduce system race conditions.

```js
try {
  fs.mkdirSync(outputDirectory);
} catch (err) {
  if (err.code !== 'EEXIST') {
    throw err;
  }
}
```

Reply:
Agree with you, I have made the changes.